frb_codegen/src/library/commands/flutter.rs

```diff
@@ -5,9 +5,19 @@ use log::info;
 use std::path::Path;
 
 #[allow(clippy::vec_init_then_push)]
-pub fn flutter_create(name: &str) -> anyhow::Result<()> {
-    info!("Execute `flutter create {name}`");
-    check_exit_code(&command_run!(call_shell[None, None], "flutter", "create", name)?)
+pub fn flutter_create(name: &str, org: &Option<String>) -> anyhow::Result<()> {
+    match org {
+        Some(o) => {
+            info!("Execute `flutter create  --org {o} {name}`");
+            check_exit_code(
+                &command_run!(call_shell[None, None], "flutter", "create", "--org", o, name )?,
+            )
+        }
+        None => {
+            info!("Execute `flutter create {name}`");
+            check_exit_code(&command_run!(call_shell[None, None], "flutter", "create", name )?)
+        }
+    }
```

Comment:
nit: Well... I mean, it would be great to be something like 

```
let args = [...];
if org {
 args += ["--org", org];
}
info!("... args");
command_run!(..., args);
```

Reply:
Thanks for your suggestions, the code is much cleaner now.